templates/zerver/help/configure-message-editing-and-deletion.md

```diff
@@ -14,22 +14,23 @@ is highly configurable. Two things are true under any configuration:
 * Any message can be deleted at any time by an organization administrator.
 
 <div class="centered_table"></div>
-|                                    | Admins   | Members |
-|---                                 |---       |---      |
-| Edit your message content          | [1]      | [1]     |
-| Edit others' message content       |          |         |
-| Edit your message topics           | [1]      | [1]     |
-| Add a topic to a topicless message | [1]      | [1]     |
-| Edit others' message topics        | [1]      | [2]     |
-| Move topics between streams        | [4]      | [4]     |
-| Delete your messages               | &#10004; | [3]     |
-| Delete others' messages            | &#10004; |         |
+|                                       | Admins   | Members |
+|---                                    |---       |---      |
+| Edit your message content             | [1]      | [1]     |
+| Edit others' message content          |          |         |
+| Edit your message topics              | [1]      | [1]     |
+| Add topic to messages without a topic | [1]      | [1]     |
+| Edit others' message topics           | [1]      | [2]     |
+| Move topics between streams           | [4]      | [4]     |
+| Delete your messages                  | &#10004; | [3]     |
+| Delete others' messages               | &#10004; |         |
 
 [1] Controlled by **Allow message editing**.
 
 [2] Controlled by **Who can edit topic of any message**.
 
-[3] Controlled by **Who can delete their own messages**.
+[3] Controlled by **Who can delete their own messages**
+and **Time limit for deleting messages**.
 
```

Comment:
Since currently [4] shows up before [3] in the table, let's swap the numbering.

Reply:
**Screenshot of updated chart and key**:
![Screenshot from 2022-03-23 13-06-44](https://user-images.githubusercontent.com/63245456/159695416-f5a3a53a-5a15-4ed4-b63b-9d604e7a6d54.png)